docs/source/TaskEngineWorkflows.md

```diff
@@ -348,6 +362,43 @@ This workflow allows you to create an MP4 from a VOD asset
 }
 ```
 
+## Build_thumbnails
+
+This workflow allows you to generate thumbnail assets which can then be used for video timeline previews.
+
+### Build_thumbnails: Parameters
+
+| Parameter Name    | Required |  Description | Default |
+| ----------------- | -------- | ------------ | ------- |
+| workflow          |Yes| Specify 'build_thumbnails'. ||
+| content_id        |Yes| Unique identifier of the content. This is usually a key that allows identification of the content in the client’s system.||
+| source            |Yes| URL of the HLS source from which to create assets (isml sources must be in a state of `stopped`). ||
+| output_folder     |Yes| This is the folder where the resulting assets wil be saved on S3| <content_id> ||
+| target_filename   |No | Prefix for the file names of generated assets, format: <target_filename>_<sprite/vtt>.jpg| <content_id>_<sprite/vtt>.jpg ||
```

Comment:
There's a problem with the format specified as 1 jpg and 1 vtt file is created. You should not include the format, saying that it is a prefix is enough. Also, set the default valut to just <content_id>

extra pipe at the end of the line